csunplugged/static/js/website.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+  $("#content-container, #glossary-modal").on("click", ".glossary-term", open_glossary_definition);
+});
+
+// Retrieve glossary definition
+function open_glossary_definition() {
+  // Save glossary slug
+  var slug = $(this).data("glossary-term");
+  // Find glossary modal
+  var glossary_modal = $("#glossary-modal");
+
+  // If glossary modal already has content for term
+  if (glossary_modal.attr("data-glossary-term") == slug) {
+    glossary_modal.modal('show');
+  }
+  // Otherwise download HTML of modal and add to DOM
+  else {
+    // TODO: Allow URL to work for different languages
```

Comment:
Both, in the past (for example: resource generation) I create TODOs when developing to find the problem area easier, and also as a reminder to create the issue. Will list it in the PR.